content/helloworld-fromcftok8s.md

```diff
@@ -178,27 +178,29 @@ Now I can deploy this with
 
-The first thing you'll run across in many examples is the concept of a Kubernetes *Service*. The [Kubernetes documentation on Services](https://kubernetes.io/docs/concepts/services-networking/service/) says a Service is "An abstract way to expose an application running on a set of Pods as a network service." The key idea of a service is that it hides the ephemeral nature of a pod - no matter how often pods get restarted or scaled up and down, the service endpoint remains accessible and doesn't change. It also performs basic (round-robin) load balancing, i.e. it makes sure that incoming requests are distributed evenly across a number of identical containers in a pod or a set of pods in a ReplicaSet. 
+Fair enough, so what else have we got? 
 
-So is a Service the Kubernetes counterpart of a Cloud Foundry application route? Not necessarily. Since there are different service types you can specify. To make our service externally reachable, we need to make it be of the type "NodePort". This essentially makes Kubernetes assign a random TCP port in the port range between 30000 and 32767 and makes this port reachable outside the cluster. There's another Service type called "LoadBalancer" which essentially implements the service by requesting a load balancer resource from the underlying cloud provider - but let's leave that aside for now. 
+The first thing you'll run across in many examples is the concept of a Kubernetes *Service*. The [Kubernetes documentation on Services](https://kubernetes.io/docs/concepts/services-networking/service/) says a Service is "an abstract way to expose an application running on a set of Pods as a network service." The key idea of a Service is that it hides the ephemeral nature of a pod - no matter how often pods get restarted or scaled up and down, the service endpoint remains accessible and doesn't change. It also performs basic (round-robin) load balancing, i.e. it makes sure that incoming requests are distributed evenly across a number of identical containers in a pod or a set of pods in a ReplicaSet. 
 
-Btw, a good article that helped me wrap my head around all this is ["Kubernetes Ingress for Beginners"](https://thenewstack.io/kubernetes-ingress-for-beginners/) by Nick Ramirez. 
+So is a Service the Kubernetes counterpart of a Cloud Foundry application route? Not necessarily, since there are different service types you can specify. To make our service externally reachable, we need to make it be of the type "NodePort." This essentially makes Kubernetes assign a random TCP port in the port range between 30000 and 32767 and makes this port reachable outside the cluster. There's another Service type called "LoadBalancer" which essentially implements the service by requesting a load balancer resource from the underlying cloud provider - but let's leave that aside for now. 
```

Comment:
```suggestion
So is a Service the Kubernetes counterpart of a Cloud Foundry application route? Not necessarily, since you can specify different service types. To make our service externally reachable, we need to make it be of the type "NodePort." This essentially makes Kubernetes assign a random TCP port in the port range between 30000 and 32767 and makes this port reachable outside the cluster. There's another Service type called "LoadBalancer" which essentially implements the service by requesting a load balancer resource from the underlying cloud provider - but let's leave that aside for now. 
```